### PR TITLE
Add task to clean old errors

### LINKED ISSUE #28: Add rake task to clean old errors

### PR DESCRIPTION
**Pull request by brunvez**
Closes #28

### CODE REVIEW

File: lib/exception_hunter/error_reaper.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module ExceptionHunter
+  class ErrorReaper
+    DELETE_UNTIL = 1.month.ago
```
Comment (t-romani) [inline lib/exception_hunter/error_reaper.rb:3]: maybe this could be configurable by the user?

File: spec/error_reaper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+module ExceptionHunter
```
Comment (t-romani) [inline spec/error_reaper_spec.rb:1]: i feel like this was just thrown inside spec? maybe it could be located somewhere else

Comment (brunvez) [reply spec/error_reaper_spec.rb:1]: Yeah but I'd like to have a good name for it. Right now there's no name that encapsulates all these modules

File: README.md [hunk 3]
```diff
@@ -32,6 +32,19 @@ you can run the command with the `--skip-users` flag.
 Additionally it should add the 'ExceptionHunter.routes(self)' line to your routes, which means you can go to
 `/exception_hunter/errors` in your browser and start enjoying some good old fashioned exception tracking!
 
+## Stale data
+
+You can get rid of stale errors by running the rake task to purge them:
+
+```bash
+$ rake exception_hunter:purge_errors
```
Comment (t-romani) [inline README.md:40]: nice

**Pull request merged**